app/pods/components/player-csv-leaderboard/template.hbs

```diff
@@ -0,0 +1,19 @@
+<div class="csv_leaderboard">
```

Comment:
don't render anything unless we have data.
see: http://api.emberjs.com/ember/3.8/classes/Component/properties/isVisible?anchor=isVisible